### ISSUE TITLE
Create Playlist from downloaded content

### ISSUE THREAD

**Issue by PoliSte** [TeamNewPipe/NewPipe#3188]
I think implementing this feature is very important
because the application would also become a music player and this could increase the downloads.

Basically you should put the possibility to create playlists with the downloaded audio.

After this the application would be perfect

**Comment by Stypox** [comment]
Can't you use a music player alltoghether?

**Comment by RickeyJLancaster** [comment]
@Stypox being able to play downloaded content in background would be really great. The users wouldn't need install another app just for playing, keep changing between apps to play and download is a bit inconvenient, users would save some storage, and I imagine that wouldn't be a complicated feature to implement, since there is already a background player on the app

**Comment by opusforlife2** [comment]
Depends on #478.

**Comment by SameenAhnaf** [comment]
@opusforlife2 Out of scope as #478 is rejected.
